pynecone/config.py

```diff
@@ -173,6 +174,12 @@ class Config(Base):
     # Additional frontend packages to install.
     frontend_packages: List[str] = []
 
+    # Enable the admin dash.
+    enable_admin: bool = False
+
+    # The Admin Dash
+    admin_dash: Optional[AdminDash] = None
```

Comment:
I think we can combine these two fields into one. If the `admin_dash` is set to `None` we can disable it.